### ISSUE TITLE
lsd wildcard matching doesn't seem to work in some cases.

### ISSUE THREAD

**Issue by dalecurtis** [Peltoche/lsd#231]
## Expected behavior (as seen in lsd directory)
```
$ ls -al *.md
-rw-r--r-- 1 dalecurtis primarygroup 5468 May 23 05:08 README.md

$ ls -al README*
-rw-r--r-- 1 dalecurtis primarygroup 5468 May 23 05:08 README.md
```

## Actual behavior
```
$ ./lsd -al *.md
$ ./lsd -al README*
```

**Comment by meain** [comment]
What is your lsd version? We had this bug in an old version and a fix for it was released recent, could you maybe update lsd and try as well.

**Comment by Peltoche** [comment]
Hi @dalecurtis,

As said by @meain a regression have been present inside the version 0.15.0 and have been fixed inside the 0.15.1. You should check you version and upgrade if necessary.

**Comment by dalecurtis** [comment]
Ah thanks. I didn't notice that there was a new version. I was on 0.15, this is now fixed. Thanks!
